src/format/svg/SVGData.hx

```diff
@@ -581,9 +581,9 @@ class SVGData extends Group {
 		path.stroke_alpha = getFloatStyle ("stroke-opacity", inPath, styles, 1.0);
 		path.stroke_colour = getStrokeStyle ("stroke", inPath, styles, null);
 		path.stroke_width = getFloatStyle ("stroke-width", inPath, styles, 1.0);
-		path.stroke_caps = getStyleAndConvert ("stroke-linecap", inPath, styles, null, 
+		path.stroke_caps = getStyleAndConvert ("stroke-linecap", inPath, styles, CapsStyle.NONE, 

```

Comment:
https://www.w3.org/TR/SVG/painting.html#StrokeLinecapProperty seems to indicate that `butt` is the default value